androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/surveydetail/Spinner.kt

```diff
@@ -0,0 +1,46 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.screen.surveydetail
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import co.nimblehq.avishek.phong.kmmic.android.ui.theme.ApplicationTheme
+import co.nimblehq.avishek.phong.kmmic.presentation.uimodel.AnswerUiModel
+import com.chargemap.compose.numberpicker.ListItemPicker
+
+@Composable
+fun Spinner(
+    modifier: Modifier = Modifier,
+    answerUiModels: List<AnswerUiModel>,
+) {
+    val answerTexts = answerUiModels.map { it.text }
+    var answerText by remember { mutableStateOf(answerTexts[0]) }
+
+    ListItemPicker(
```

Comment:
I didn't know this exist. That's really neat 👍 But can we reduce the opacity for the unelected ones? They are currently nearly invisible. And also can you please try to reduce the thickness of the separator?

Reply:
I used a 3rd-party library for this. Unfortunately, it doesn't allow hanging the divider thickness and the unselected text styles. We might need to modify the library implementation for our needs. 🤔 How about we do this in a different task?